src/components/locales/ar/translation.json

```diff
@@ -1,3 +1,3 @@
 {
-  "omar": "عمر"
-}
+  "We Care About Your Health": "صحتكم تهمنا"
```

Comment:
the key should be a short name like "title" or something, not the english version of the sentence